recipes/ctat-lncrna/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "ctat-lncrna" %}
+{% set version = "2.0" %}
+{% set build = "1" %}
```

Comment:
should be 0 if you have a new version

Reply:
I am making a new build including the changes you suggested.